src/main/views/lists/tabs/contactTypes/editContactType.njk

```diff
@@ -0,0 +1,71 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% set isNewContactType = true %}
+{% if contactType.id %}
+  {% set isNewContactType = false %}
+{% endif %}
+
+{% if isNewContactType %}
+  <h2 class="govuk-heading-l">Add New Contact Type</h2>
+{% else %}
+  <h2 class="govuk-heading-l">Editing Contact Type: {{contactType.type}}</h2>
+{% endif %}
+
+{% if errors|length %}
+  {{ govukErrorSummary({
+    titleText: "There is a problem",
+    errorList: errors
+  }) }}
+{% endif %}
+
+{% if fatalError == false %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        {{ govukInput({
+          label: {
+            text: "Name"
+          },
+          id: "ct-type",
+          name: "contactType[type]",
+          value: contactType.type,
+          errorMessage: { text: 'Name is required' } if nameValid == false
+        }) }}
+
+
+
+
+      <input id='contactTypeId' type='hidden' name='contactType[id]' value={{contactType.id}}></input>
```

Comment:
Should we be putting the id in a hidden field? I imagine this would still show it on the frontend if someone were to inspect the html? 

Is there a way to send this through as data instead?

Reply:
needed for javascript when editing existing contact